scratch-vm/src/extension-support/extension-manager.js

```
@@ -332,19 +332,21 @@ class ExtensionManager {
+            createWorker = () => Promise.resolve(new ExtensionWorker());
         } else if (sandboxMode === 'iframe') {
-            ExtensionWorker = (await import(/* webpackChunkName: "iframe-extension-worker" */ './tw-iframe-extension-worker')).default;
+            const IframeExtensionWorker = (await import(/* webpackChunkName: "iframe-extension-worker" */ './tw-iframe-extension-worker')).default;
+            createWorker = () => IframeExtensionWorker.create();
         } else {
             throw new Error(`Invalid sandbox mode: ${sandboxMode}`);
         }
         /* eslint-enable max-len */
 
         return new Promise((resolve, reject) => {
            this.pendingExtensions.push({ extensionURL, rewrittenURL: rewritten, resolve, reject });
```

ESLint

[error] There should be no space after '{'.

(object-curly-spacing)